packages/create-sitecore-jss/src/templates/nextjs-sxa/src/assets/sass/components/_component-image.scss

```diff
@@ -9,7 +9,7 @@
         display: inline-block;
         max-width: 100%;
     }
-    .image-caption {
+    span {
```

Comment:
@matkovskyi Would you mind please also making the same change here?
https://github.com/Sitecore/jss/blob/dev/packages/create-sitecore-jss/src/templates/angular-xmcloud/src/assets/styles/sass/components/_component-image.scss#L12

For now, we have duplicated the styles in Angular, so will need maintain both.